core/src/main/resources/jenkins/util/JenkinsIsSafeRestarting/index.jelly

```diff
@@ -0,0 +1,68 @@
+<!--
```

Comment:
After the cleanup (thanks!) these resource files are unnecessary.

Reply:
Ah, now I remember why I had the extra class. Because the jelly content is different. I will add a conditional jelly section